CHANGELOG.md

```diff
@@ -5,6 +5,21 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- **9** new parameters to the `logger` function as below and log them too:
```

Comment:
Added section is for new features, format, or file.
And also logger is an inner function, only add a general sentence in changed section, for example :
- `logger` function modified

Reply:
Fixed